packages/SwingSet/src/controller.js

```diff
@@ -37,6 +43,51 @@ function makeConsole(tag) {
   return harden(cons);
 }
 
+function makeStartXSnap(bundles, { env }) {
+  const xsnapOpts = {
+    os: osType(),
+    spawn,
+    stdout: 'inherit',
+    stderr: 'inherit',
+    debug: !!env.XSNAP_DEBUG,
+  };
+
+  const pool = defaultSnapstorePath({ env });
```

Comment:
This default makes me uncomfortable, and I'm sure the use of `env` will drive MarkM out of his mind. Let's require callers to provide a path. We'll need to add an option to `makeSwingsetController` to pass it through (maybe `runtimeOptions.snapstorePath`), but I think it's ok for `makeStartXSnap` to create the indicated directory if it doesn't already exist.

Reply:
Ok... so this means changing all the callers... i.e. all the main programs and roughly all the tests. Do you have a suggestion for what path they should use?

Reply:
ah... I can skip the snapstore if no snapstorePath is provided